src/main/java/com/tieumy/exercise/NumeralEx03.java

```diff
@@ -0,0 +1,21 @@
+package com.tieumy.exercise;
+
+public class NumeralEx03 {
+
+    public static String factorizeNumber(final int n) {
+        int enteredNumber = n;
+        String result = "";
+        for (int i = 2; i <= enteredNumber; i++) {
+            while (enteredNumber % i == 0) {
+                enteredNumber = enteredNumber / i;
+                if (enteredNumber == 1)
+                    result = result + (i + "");
+                else
+                    result = result + (i + " * ");
+            }
+            if (enteredNumber == 1)
```

Comment:
Câu lệnh này không cần thiết vì không có nó thì vòng for vẫn chấm dứt bình thường

Reply:
Done